Applications/LLaMA/jni/main.cpp

```diff
@@ -393,7 +415,7 @@ void createAndRun(unsigned int epochs, unsigned int batch_size) {
   // model->summarize(std::cout, ML_TRAIN_SUMMARY_MODEL);
 
   std::string weight_path =
-    optimize ? "./llama_v2_att.bin" : "./summarization_v2_fp16.bin";
+    optimize ? "./llama_v2_att.bin" : "/home/donghak/Desktop/llama_v2.bin";
```

Comment:
Please remove the absolute path (/home/donghak/Desktop/) here.

Reply:
Fix, Thank you for Review